guardian/dotcom-rendering
# Support new List element fields in `apps-rendering`

**Note: this is currently blocked while we resolve the issue which is preventing the latest version of `apps-rendering-api-models` from being published (https://github.com/guardian/apps-rendering-api-models/pull/87)**

## What does this change?

Supports the new list element fields in elements in `apps-rendering`

## Why?

If we don't make this change, Mini profile articles will not correctly render in AR and therefore editions

## How to test

To test this change, we'll need to:

- publish a mini profiles article on CODE
- verify the article doesn't appear correctly here: 
https://mobile.code.dev-guardianapis.com/uk/rendered-items/PATH-GOES-HERE
- update configuration to point CODE MAPI to CODE Fronts and CODE CAPI. This can be done by running [this script](https://github.com/guardian/mobile-platform/blob/main/scripts/mapi-code-config.sh) with MAPI Janus credentials, or by asking someone nicely in the apps team to do it
- redeploy `mobile-fronts` and `mobile-item` to CODE so the config changes took effect
- deploy this branch (`apps-rendering-new-list-fields`) to CODE (project name: `Mobile::mobile-apps-rendering`)
- verify the mini profiles article _does_ render correctly here:
https://mobile.code.dev-guardianapis.com/uk/rendered-items/PATH-GOES-HERE
it should include the new fields `bio` and `endNote`
